### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by maartenvn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,51 +9,40 @@
...
-            "autoprefixer": {}
-        }
+        "vue-cli-plugin-vuetify": "^0.6.3",
+        "vue-template-compiler": "^2.6.10",
+        "vuetify-loader": "^1.2.2"
     },
     "browserslist": [
         "> 1%",
-        "last 2 versions",
-        "not ie <= 8"
```
Comment (Jan-PieterBaert) [inline package.json]: Why is this deleted?

File: src/App.vue [hunk 2]
```diff
@@ -1,80 +1,29 @@
...
+import Footer from "@/components/layout/Footer.vue";
 
 export default {
-  name: "App",
-  components: {
-    Navigation,
-    Footer
-  },
-  computed: {
-    isDark() {
```
Comment (Jan-PieterBaert) [inline src/App.vue:26]: Don't forget to add this (later) to the rewrite

**Review by Jan-PieterBaert — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 2 above)
Comment (maartenvn) [reply src/App.vue:26]: Will add this in the next commit.

File: package.json (hunk 1 above)
Comment (maartenvn) [reply package.json]: Will add this in the next commit.

**Review by Jan-PieterBaert — APPROVED** [review]
